keps/sig-apps/4017-pod-index-label/README.md

```diff
@@ -411,7 +411,8 @@ existing label which other things may depend on, for example).
 - Docs are clear about what happens if two pods get the same value (it is set by workload controllers, nothing in the API system will prevent collisions from happening).
 
 #### GA
-Fix any potentially reported bugs.
+- Fix report doc bug: https://github.com/kubernetes/kubernetes/pull/125502/
+- Add IndexJob pod label (same as existing annotation)
```

Comment:
I see this code:

```
func addCompletionIndexLabel(template *v1.PodTemplateSpec, index int) {
	if template.Labels == nil {
		template.Labels = make(map[string]string, 1)
	}
	// For consistency, we use the annotation batch.kubernetes.io/job-completion-index for the corresponding label as well.
	template.Labels[batch.JobCompletionIndexAnnotation] = strconv.Itoa(index)
}
```

I think IndexJob pod label has already been added in beta.

Reply:
thanks for checking, removed from the list